### DISCUSSION TITLE
Raudio Android app no longer available for Android version 12

### DISCUSSION THREAD

**Discussion by githans1** [rern/rAudio#1474] [Bug Reports]
I accidentally deleted my raudio app from my phone running Android version 12. 

Reinstalling seems not possible since app available in the Google store is not compatible with my version of Android. 

Are there still old apps around which are compatible (my phone no longer allows upgrading the Android version)?

**Comment by AndyFx12** [comment]
A downloadable copy of original APK is here:
https://cloud.s-t-franz.de/s/SG82dQHRr27sjFS

App works even on Android 13.

regards, Andy

**Comment by githans1** [comment]
Thanks Andy



Get TypeApp for Android<http://www.typeapp.com/r>
On Oct 30, 2023, at 10:28 AM, Andreas Franz ***@***.******@***.***>> wrote:

A downloadable copy of original APK is here:
https://cloud.s-t-franz.de/s/SG82dQHRr27sjFS

App works even on Android 13.

regards, Andy

**Comment by mutlucan96** [comment]
![Screenshot from 2023-11-05 18-01-49](https://github.com/rern/rAudio/assets/42113538/faa7af3d-70e0-474a-961f-ed5418803991)
It is not available for any of my devices with different versions of Android. 7, 10, 11, 12

**Reply by mutlucan96** [reply]
An apk file would be also nice :)

**Reply by AndyFx12** [reply]
see: https://github.com/rern/rAudio/discussions/1474#discussioncomment-7421702

**Reply by mutlucan96** [reply]
Oh, sorry and thanks :)

**Comment by zyglob** [comment]
The app seems to have been removed from the Google Play storefront but can be sideloaded from the link Andy has given. Thanks, Andy.

I suspect that the app doesn't meet Google's required API level and needs to be updated.

**Comment by cmh714** [comment]
there is an adb command I believe that will allow installation for these type of apps. Cant recall if root is required....
